packages/owl-bot/src/create-pr.ts

```diff
@@ -70,7 +70,16 @@ export async function createPullRequestFromLastCommit(
   const githubRepo = await octokit.repos.get({owner, repo});
 
   cmd(`git remote set-url origin ${pushUrl}`, {cwd: localRepoDir});
-  cmd(`git push origin ${branch}`, {cwd: localRepoDir});
+
+  try {
+    cmd(`git push origin ${branch}`, {cwd: localRepoDir});
+  } catch (e) {
+    logger.info('Falling back to pull, push', e);
+
+    // In case we're missing commits from an existing branch
+    cmd(`git pull origin ${branch}`, {cwd: localRepoDir});
```

Comment:
Question.

Sometimes the PR author force pushing to the PR head branch. What will happen in such cases?

Reply:
That case would be handled in the follow flow: the `push` in the try block would fail, which we would then attempt to `pull` then `push`

Reply:
@SurferJeffAtGoogle 

I think when this code runs, the local git repo may have some changes.
If the remote branch is force pushed and containing different history, it may fail.

WDYT?